src/main/java/seedu/address/ui/CustomerCard.java

```diff
@@ -23,7 +23,7 @@
      * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX. As a consequence, UI
      * elements' variable names cannot be set to such keywords or an exception will be thrown by JavaFX during runtime.
      *
-     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on ContactList level 4</a>
```

Comment:
I think this one and this one alone should be unchanged given that it references an external project